lib/elixir/lib/exception.ex

```diff
@@ -177,6 +177,16 @@ defmodule Exception do
     end
   end
 
+  @spec format(String.t(), any) :: String.t()
+  def format_message_with_term(message, term) do
+    inspected =
+      term
+      |> inspect(pretty: true)
+      |> String.replace(~r/^(?=.+)/m, "    ")
```

Comment:
You should be able to achieve this by splitting on new lines and indenting every non empty string.

Reply:
We should add padding option to inspect at some point.